src/libstd/sys/windows/args.rs

```diff
@@ -48,14 +185,13 @@ impl<'a> fmt::Debug for ArgsInnerDebug<'a> {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         f.write_str("[")?;
         let mut first = true;
-        for i in self.args.range.clone() {
+        for i in self.args.parsed_args_list.as_slice() {
```

Comment:
Sorry, to clarify, I think this entire function can be replaced with `self.args.parsed_args_list.as_slice().fmt(f)`